Cargo.toml

```diff
@@ -7,9 +7,9 @@ members = [
   "quickwit-cluster",
   "quickwit-common",
   "quickwit-config",
-  "quickwit-core",
```

Comment:
`quickwit-core` did not has really functionnal meaning so removing it is a good thing as now we have a place where we can put all stuff relative to index operations (control plane).